.github/workflows/examples-java-push.yml

```diff
@@ -92,5 +92,5 @@ jobs:
           cf target -o "${CF_ORG}" -s "${CF_SPACE}"
       - name: Deploy to PaaS
         run: |
-          cf push -f manifest.yml --strategy rolling
+          cf push -f manifest.yml -p examples/java/spring-boot/build/libs/federated-api-model.jar --strategy rolling
```

Comment:
Is it possible to check this on a branch, i.e. creating a new job before deploy, which depends on `build` and copies the lib? (Similar to the changes I was doing yesterday on DSA Workbench, this is one that we can check more easily, as it's GitHub Actions related)

Reply:
Sorry I don't think I was clear when we spoke offline - I'd be expecting to see i.e. `ls -al` and `ls -al examples/java/spring-boot/build/libs/federated-api-model.jar` to verify that the path is actually there. Right now we're seeing if we can download the artefact, but not that it'll be present in the location we think it'll be in